### PR TITLE
DLS-1393 GMP Bootstrap Upgrade - Frontend

### PR DESCRIPTION
**Pull request by GujjalapudiA**
Remove Play global settings and update configuration/structure based on the guide.

Catalogue update

### CODE REVIEW

File: app/config/GmpModule.scala [hunk 1]
```diff
@@ -16,18 +16,23 @@
...
   override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = Seq(
-    bind[AuthConnector].to(classOf[GmpAuthConnector]),
-    bind[HttpGet].to(WSHttp),
-    bind[HttpPost].to(WSHttp),
-    bind[HttpPut].to(WSHttp),
-    bind[WSHttp].to(WSHttp)
+    bind(classOf[HttpClient]).to(classOf[DefaultHttpClient]),
+      bind(classOf[AuthConnector]).to(classOf[DefaultAuthConnector]),
+      bind(classOf[AuditConnector]).to(classOf[DefaultAuditConnector]),
+      bind[HttpGet].to(WSHttp),
```
Comment (calinursaciuc) [inline app/config/GmpModule.scala]: Remove all HTTP GET/POST/PUT

File: app/config/wiring.scala [hunk 2]
```diff
@@ -28,11 +28,12 @@ import uk.gov.hmrc.play.audit.http.HttpAuditing
 import uk.gov.hmrc.play.audit.http.config.AuditingConfig
```
Comment (calinursaciuc) [inline app/config/wiring.scala]: Delete this file

File: app/config/ApplicationGlobal.scala [hunk 3]
```diff
@@ -16,53 +16,53 @@
 
 package config
```
Comment (calinursaciuc) [inline app/config/ApplicationGlobal.scala]: Delete this file and only create a MyErrorHandler without injecting things that we don't need. Look in cbcr-frontend

File: app/config/wiring.scala [hunk 4]
```diff
@@ -54,7 +55,7 @@ object WSHttp extends WSHttp with Hooks
 
 class GmpSessionCache @Inject()(environment: Environment,
```
Comment (calinursaciuc) [inline app/config/wiring.scala]: Only keep the GmpSessionCache and rename this file to match class name...Delete everything else

**Comment by GujjalapudiA** [comment]
updated as per comments

File: app/config/MyErrorHandler.scala [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
+@Singleton
+class MyErrorHandler @Inject()(
+                                val messagesApi: MessagesApi, val configuration: Configuration,
+                                val gmpContext:GmpContext
+                              ) extends FrontendErrorHandler {
+
+
+  override def standardErrorTemplate(pageTitle: String, heading: String, message: String)
+                                    (implicit request: Request[_]) = ??? // put the code from your old global object here.
```
Comment (calinursaciuc) [inline app/config/MyErrorHandler.scala]: Are we supposed to have some implementation here...check cbcr-frontend

**Pull request merged**